src/styles/_catalog.scss

```diff
@@ -1,22 +1,25 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(auto-fill, 200px);
 
   max-width: 944px;
   margin: 0 auto;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
+    display: grid;
+    place-items: center;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
   }
+}
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+@media (max-width: 488px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: 200px;
   }
 }
```

Comment:
Just move it inside `.catalog`
BTW there should still be 2 columns on 488px
![image](https://user-images.githubusercontent.com/99403198/203974797-89298a7f-c7d1-4b68-b264-ec94437bbacd.png)
![image](https://user-images.githubusercontent.com/99403198/203974831-6f9602ed-c9fe-406d-8dc2-f8021e6f4f65.png)